metric_providers/psu/energy/ac/ipmi/machine/ipmi-get-machine-energy-stat.sh

```diff
@@ -2,15 +2,35 @@
 set -euo pipefail
 
 i=''
-
-while getopts "i:" o; do
+check_system=False
```

Comment:
Captilalied in Bash? isn't it false ?

Reply:
that's true... it was working before, but in a string comparision way. Rewrote both of the shell scripts which use this to use bash's true/false checks